packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -1,1093 +1,717 @@
+	"runData.executionTime": "Execution Time",
+	"runData.fileExtension": "File Extension",
+	"runData.fileName": "File Name",
+	"runData.items": "Items",
+	"runData.json": "JSON",
+	"runData.mimeType": "Mime Type",
+	"runData.ms": "ms",
+	"runData.noBinaryDataFound": "No binary data found",
+	"runData.noData": "No data",
+	"runData.noTextDataFound": "No text data found",
+	"runData.nodeReturnedALargeAmountOfData": "Node returned a large amount of data",
+	"runData.output": "Output",
+	"runData.downloadBinaryData": "Download",
+	"runData.showBinaryData": "View",
+	"runData.startTime": "Start Time",
+	"runData.table": "Table",
+	"runData.theNodeContains": "The node contains {numberOfKb} KB of data.<br />Displaying it could cause problems.<br /><br />If you do decide to display it, consider avoiding the JSON view.",
+	"saveButton.save": "@:reusableBaseText.save",
+	"saveButton.saved": "Saved",
+	"saveButton.saving": "Saving",
+	"showMessage.cancel": "@:reusableBaseText.cancel",
+	"showMessage.ok": "OK",
+	"showMessage.showDetails": "Show Details",
+	"tagsDropdown.createTag": "Create tag \"{filter}\"",
+	"tagsDropdown.manageTags": "Manage tags",
+	"tagsDropdown.noMatchingTagsExist": "No matching tags exist",
+	"tagsDropdown.noTagsExist": "No tags exist",
+	"tagsDropdown.showError.message": "A problem occurred when trying to create the '{name}' tag",
+	"tagsDropdown.showError.title": "Could not create tag",
+	"tagsDropdown.typeToCreateATag": "Type to create a tag",
+	"tagsManager.couldNotDeleteTag": "Could not delete tag",
+	"tagsManager.done": "Done",
+	"tagsManager.manageTags": "Manage tags",
+	"tagsManager.showError.onCreate.message": "A problem occurred when trying to create the tag '{escapedName}'",
+	"tagsManager.showError.onCreate.title": "Could not create tag",
+	"tagsManager.showError.onDelete.message": "A problem occurred when trying to delete the tag '{escapedName}'",
+	"tagsManager.showError.onDelete.title": "Could not delete tag",
+	"tagsManager.showError.onUpdate.message": "A problem occurred when trying to update the tag '{escapedName}'",
+	"tagsManager.showError.onUpdate.title": "Could not update tag",
+	"tagsManager.showMessage.onDelete.title": "Tag deleted",
+	"tagsManager.showMessage.onUpdate.title": "Tag updated",
+	"tagsManager.tagNameCannotBeEmpty": "Tag name cannot be empty",
+	"tagsTable.areYouSureYouWantToDeleteThisTag": "Are you sure you want to delete this tag?",
+	"tagsTable.cancel": "@:reusableBaseText.cancel",
+	"tagsTable.createTag": "Create tag",
+	"tagsTable.deleteTag": "Delete tag",
+	"tagsTable.editTag": "Edit Tag",
+	"tagsTable.name": "@:reusableBaseText.name",
+	"tagsTable.noMatchingTagsExist": "No matching tags exist",
+	"tagsTable.saveChanges": "Save changes?",
+	"tagsTable.usage": "Usage",
+	"tagsTableHeader.addNew": "Add new",
+	"tagsTableHeader.searchTags": "Search Tags",
+	"tagsView.inUse": "{count} workflow | {count} workflows",
```

Comment:
love that I can simply search for this now

Reply:
:)